packages/flutter_tools/templates/module/android/library/include_flutter.groovy.copy.tmpl

```diff
@@ -6,16 +6,18 @@ def flutterProjectRoot = new File(scriptFile).parentFile.parentFile
 gradle.include ':flutter'
 gradle.project(':flutter').projectDir = new File(flutterProjectRoot, '.android/Flutter')
 
-def plugins = new Properties()
-def pluginsFile = new File(flutterProjectRoot, '.flutter-plugins')
-if (pluginsFile.exists()) {
-    pluginsFile.withReader('UTF-8') { reader -> plugins.load(reader) }
-}
+if (System.getenv('ENABLE_FLUTTER_BUILD_PLUGINS_AS_AAR') != 'true') {
```

Comment:
Ditto - if this can just be another -P parameter somehow

Reply:
that won't automatically work in add 2 app.

Reply:
project properties aren't available at this time in Gradle. AAR plugins won't be available in `include_flutter.groovy` at this time.. will follow later cc @xster.

Reply:
this is generated now in the template.